sklearn/model_selection/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .split import (KFold, StratifiedKFold, LeaveOneLabelOut, LeaveOneOut,
```

Comment:
I know we're not being terribly consistent about this but I'd rather do one import per line as in [the metrics init](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/__init__.py), if it's not too tedious (some IDEs should be able to do it)

Makes diffs better in the future.